components/movementsensor/collectors.go

```diff
@@ -4,12 +4,42 @@ import (
 	"context"
 	"errors"
 
+	v1 "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/movementsensor/v1"
 	"google.golang.org/protobuf/types/known/anypb"
 
 	"go.viam.com/rdk/data"
 )
 
 // TODO: add tests for this file.
```

Comment:
Do we have a ticket for this TODO? (I know this was already here but figured I would ask 🙂)

Reply:
I will make this ticket and update the comment.